agent/src/main/java/com/jonnyzzz/teamcity/plugins/node/agent/nvm/NVM.kt

```diff
@@ -105,11 +106,19 @@ class NVMRunner(val downloader : NVMDownloader,
           downloader.downloadNVM(nvmHome, url)
           message("NVM downloaded into $nvmHome")
         }
+        
         script("Install", "Installing Node.js v$version",nvmHome.path) {
-          ". $nvmHome/nvm.sh" n "nvm install $fromSource $version"
+          if (isWindows)
+            "nvm install $fromSource $version"
+          else
+            ". $nvmHome/nvm.sh" n "nvm install $fromSource $version"
```

Comment:
it is better to reuse the same part of the command

Reply:
👍